### PR TITLE
[FLINK-25179] Add document for composite types support for parquet

### PR DESCRIPTION
**Pull request by meetjunsu**
## What is the purpose of the change

Add document for array,map,row types support for parquet row writer.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / **no**)
  - The serializers: (yes / **no** / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / **no** / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (yes / **no** / don't know)
  - The S3 file system connector: (yes / **no** / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / **no**)

### CODE REVIEW

**Comment by meetjunsu** [comment]
Related PR: [https://issues.apache.org/jira/browse/FLINK-17782](url)

**Comment by meetjunsu** [comment]
@wuchong  Could you help to review this pr?

File: docs/content/docs/connectors/table/formats/parquet.md [hunk 1]
```diff
@@ -174,9 +174,24 @@ The following table lists the type mapping from Flink type to Parquet type.
...
+      <td>ROW</td>
+      <td></td>
+      <td>STRUCT</td>
+    </tr>
     </tbody>
 </table>
 
 {{< hint warning >}}
-Composite data type: Array, Map and Row are not supported.
+Composite data type: Array, Map and Row currently only support writing but not reading.
```
Comment (MartijnVisser) [inline docs/content/docs/connectors/table/formats/parquet.md:196]:
```suggestion
Composite data type: Array, Map and Row are currently only supported when writing, not reading.
```

**Review by MartijnVisser — COMMENTED** [review]
Thanks for the PR @meetjunsu - Could you fix the minor thing for the English translation, then I can approve it and find someone to merge it.

**Comment by meetjunsu** [comment]
> Thanks for the PR @meetjunsu - Could you fix the minor thing for the English translation, then I can approve it and find someone to merge it.

@MartijnVisser  Thanks for your suggestion, it has been updated.

**Review by MartijnVisser — APPROVED** [review]
Thanks for fixing it, LGTM.
@fapaul Can you review & merge this?

**Comment by fapaul** [comment]
@meetjunsu can you squash your commits together and rebase with the latest master?

**Comment by meetjunsu** [comment]
> @meetjunsu can you squash your commits together and rebase with the latest master?

done

**Comment by MartijnVisser** [comment]
@meetjunsu Thanks for that. @fapaul should be OK now

**Pull request merged**